onnxscript/backend/onnx_export.py

```diff
@@ -230,6 +212,28 @@ def _rename_variable_s(self, name):
         """Renames all names equal to a python keyword."""
         return str(self._rename_variable(name))
 
+    def _rename_domain(self, domain: str) -> str:
+        if domain == "":
+            return "opset"
+        return domain.replace(".", "_")
```

Comment:
Move this method out to the global namespace as it doesn't depend on `self`?

Reply:
It doesn't use self yet. But, logically, it may need other state information (to avoid name clashes). I will leave it in its current place for now. Also, I am making minimal changes for now, as this is likely to undergo more significant changes (like elimination of templates) (re. https://github.com/microsoft/onnxscript/pull/873 )